lib/datasets/adult.rb

```diff
@@ -0,0 +1,68 @@
+require "csv"
+
+require_relative "dataset"
+
+module Datasets
+  class Adult < Dataset
+    Record = Struct.new(
+      :age,
+      :workclass,
+      :fnlwgt,
+      :education,
+      :education_num,
+      :marital_status,
+      :occupation,
+      :relationship,
+      :race,
+      :sex,
+      :capital_gain,
+      :capital_loss,
+      :hours_per_week,
+      :native_country,
+      :income_per_year
+    )
+
+    def initialize
```

Comment:
Can you also support `adult.test` data like `MNIST` does?

Reply:
Sure. I'm working on it. It may take some more time.